src/main/java/com/example/application/ui/horizontal/projects/NordlichtView.java

```diff
@@ -25,16 +22,18 @@
 /**
  *  Nordlicht View shows ...
  *
- *  @author Monika Martius and Laura Neuendorf
- *  @version 2.0
+ *  @author Monika Martius and Laura Neuendorf, Jessica Reistel
```

Comment:
Authors are those who created the files/classes, please do not change

Reply:
Sorry.. Habe ich zumindest bisher immer so gemacht, weil sieht man ja wenn ein neues Update kommt gar nicht mehr, wer daran alles gearbeitet hat..

Reply:
man kann die History von der einzelnen Dateien sehen

Reply:
Wusste ich nicht, aber danke